tests/test_discovery.py

```diff
@@ -42,7 +42,7 @@ def test_discovery(tmp_path):
     )
     db = databases.DuckDatabaseBackend(f"{tmp_path}/duck.db")
     cursor = db.cursor()
-    table_rows = conftest.get_sorted_table_data(cursor, "discovery__code_sources")
+    table_rows, cols = conftest.get_sorted_table_data(cursor, "discovery__code_sources")
```

Comment:
`cols` isn't used anywhere, yeah? Does ruff not complain about it? That feels like a missing check to my brain. Usually I'd prefix it with a sunder, like `_cols` to avoid warnings / signal to other devs that they can ignore it.

(Do you use the result anywhere? Is it just for manual debugging?)

Reply:
It's in the except statement on line 68.